data/json/recipes/recipe_food.json

```diff
@@ -4094,7 +4099,12 @@
     "tools": [ [ [ "surface_heat", 5, "LIST" ] ] ],
     "components": [
       [ [ "hickory_nut_shelled", 4 ] ],
-      [ [ "milk_standard_raw", 1, "LIST" ], [ "milk_powder", 1 ], [ "con_milk", 1 ] ],
+      [
+        [ "milk_standard_raw", 1, "LIST" ],
+        [ "lf_milk_standard", 1, "LIST" ],
```

Comment:
Need a source.

Reply:
https://www.tyrantfarms.com/recipe-how-to-make-hickory-nut-ambrosia/

Recipe says to add it at the very end or just let people add milk according to their taste. Supposedly if someone wouldn't want to use actual milk, they could add lactose-free milk instead.